android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/VPNServiceBinder.kt

```diff
@@ -96,7 +96,7 @@ class VPNServiceBinder(service: VPNService) : Binder() {
                 // [data] is empty
                 // Activate the tunnel with the last config
                 try {
-                    this.mService.reconnect()
+                    this.mService.reconnect(source = "system")
```

Comment:
I don't understand why this one is "system". Would you mind clarifying?

Reply:
It comes from `ACTIONS.reactivate`. The only place I see this used (from a global search on `reactivate`) is in `VPNTileService.kt`, where it's used when activating the VPN from the settings quickaction tile, which is outside the app.

Reply:
Hum, this is an interesting case. We implemented the quicksettings tile, it's not really system setting IIUC. Should we maybe add a new key here e.g. "quicksettings"? IIRC there is a similar thing we can do on iOS so it may be necessary over there as well in the future.

Reply:
The goal of the `daemon_session_source` is to figure out how many sessions are starting from the app, and how many are starting from somewhere outside the app (currently called “system”). I'm going to update the yaml file to make this more clear, and so I think keeping this just as "system" might be okay.